### ISSUE TITLE
Concussion grenade.

### ISSUE THREAD

**Issue by vda8** [studio-minus/ppg-bugs-and-requests#888]
**Feature description**
A grenade that pushes things away a lot, and has low damage and 0 shrapnel.
So like:
Grenade. small ouch, small push, small range, big shrapnel.
Dynamite. big ouch, big push, big range, no shrapnel.
Concussion grenade. small ouch, big push, big range, no shrapnel.

**Feature purpose**
To have an explosive that balances the weight between "big ouch big push" and "small ouch small push".

**Additional context**
It is epic.

**Comment by ppgplayer** [comment]
very epic i agree

**Comment by DindinYT37** [comment]
Duplicate of #608

**Comment by vda8** [comment]
> 

Nah, it's different. It's an actual grenade. Not a sack of gunpowder.

**Comment by vda8** [comment]
But is an activatable grenade. Not just a thing that goes boom when you shoot it. Smartass.

**Comment by DindinYT37** [comment]
> But it's an activatable grenade. Not just a thing that goes boom when you shoot it. Smartass.

It's the same thing, it's just one of them requires more creativity 🗿. Smartass (especially for being petty enough to react with a thumbs down to a comment lmao).

**Comment by vda8** [comment]
nobody cares :troll:

**Comment by DindinYT37** [comment]
> nobody cares :troll:

Then why do you keep answering, this is clogging the replies, I'm already hiding my own replies because I'm aware of this.
